src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -77,7 +80,13 @@ public class ProjectMapper : SnapshotMapperBase<ModelProject, SnapshotProject>
 
       public override async Task<ModelProject> MapToModel(SnapshotProject snapshot)
       {
-         var project = new ModelProject {Description = snapshot.Description};
+         var project = new ModelProject
+         {
+            Description = snapshot.Description,
+            Creation = _creationMetaDataFactory.Create()
+         };
+         project.Creation.InternalVersion = snapshot.Version;
+         project.Creation.Version = ProjectVersions.FindBy(snapshot.Version)?.VersionDisplay;
 
```

Comment:
Hard to say, if it's correct or not. 
On the one side, project is created with the **_current_** PK-Sim version and not with the **_snapshot_** version.
On the other side, if a snapshot was created with an older version of PK-Sim and contains all that noise, exporting a new snapshot from the recreated project will produce all the noise once again (and thus e.g. it would be good to get the export warning once again).

Reply:
That's the only way to track the original version however, unless we introduce yet another field for that

Reply:
>That's the only way to track the original version however, unless we introduce yet another field for that

Yep, let's keep it as is for now